inspirehep/dojson/common/base.py

```diff
@@ -190,14 +190,18 @@ def collections(self, key, value):
     """Collection this record belongs to."""
     value = force_force_list(value)
 
+    def is_deleted(value):
+        return value and value.lower() == 'deleted'
+
     def get_value(value):
-        primary = force_single_element(value.get('a'))
         return {
-            'primary': primary,
-            'secondary': value.get('b'),
-            'deleted': value.get('c'),
+            'primary': force_single_element(value.get('a')),
+            'secondary': force_force_list(value.get('b')),
         }
 
+    if not self.get('deleted', False):
+        self['deleted'] = is_deleted(value[0].get('c'))
+
```

Comment:
Yep, sorry, this should actually go inside the get_value, though it would be nicer outside... just after the appent, something like 'update_deleted' or similar